runtime/cudaq/cudaq.cpp

```diff
@@ -100,7 +100,7 @@ cudaq::MPIPlugin *getMpiPlugin(bool unsafe) {
   return g_plugin.get();
 };
 
-bool available() { return getMpiPlugin(/*unsafe=*/true) != nullptr; }
+bool available() { return static_cast<bool>(getMpiPlugin(/*unsafe=*/true)); }
```

Comment:
You shouldn't need the `static_cast` here, and I don't think it helps that much in understanding the code.